### ISSUE TITLE
On Github: a lot of missing language pages

### ISSUE THREAD

**Issue by Neustradamus** [riseupnet/riseup_help#593]
There are missing language pages, for examples (a lot of other I think):

There are:
- https://riseup.net/de/chat/clients
- https://riseup.net/en/chat/clients
- https://riseup.net/es/chat/clients
- https://riseup.net/fr/chat/clients
- https://riseup.net/pt/chat/clients
- https://riseup.net/ru/chat/clients

There are not:
- https://riseup.net/ca/chat/clients
- https://riseup.net/el/chat/clients
- https://riseup.net/hi/chat/clients
- https://riseup.net/it/chat/clients
- https://riseup.net/pl/chat/clients
- https://riseup.net/zh/chat/clients

**Comment by traumschule** [comment]
Thanks for checking! This is because no one translated them yet. Someone could find translators for those languages.

**Comment by RainerBielefeld** [comment]
I would like to help with translating to German Language, for example <https://riseup.net/de/lists/list-user/sending-mail>. Where can I find the source and some minimun instructions? 

-- 
Rainer

**Comment by micah** [comment]
RainerBielefeld <notifications@github.com> writes:

> I would like to help with translating to German Language, for example <https://riseup.net/de/lists/list-user/sending-mail>. Where can I find the source and some minimun instructions? 

Thanks for offering to help translate!

You can find the source and instructions here: https://github.com/riseupnet/riseup_help

**Comment by RainerBielefeld** [comment]
I downloaded riseup_help-master.zip
Will it be useful for you that I create a translated "de.text" from "en.text" in folder riseup_help-master?
If not, what would you recommend (please idiot proof step by step)? If we see that this is not so very easy it might be useful to agree on a Time for a chat, for me 2019-05-09  17:00 UTC would be ok via jabber. My account is rbd@jabber.de

**Comment by Neustradamus** [comment]
@RainerBielefeld, yes English is the base, you can copy and rename and edit files :)

Please verify other german files, maybe a bad word in sentence... ^^

**Comment by RainerBielefeld** [comment]
So I will contribute a first German Textfile until weekend. After Integration of that one we will see how to continue.
